homeassistant/components/totalconnect/config_flow.py

```diff
@@ -28,15 +29,16 @@ class TotalConnectConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    client: TotalConnectClient
+
     def __init__(self) -> None:
         """Initialize the config flow."""
-        self.username = None
-        self.password = None
-        self.usercodes: dict[str, Any] = {}
-        self.client = None
+        self.username: str | None = None
+        self.password: str | None = None
+        self.usercodes: dict[int, str | None] = {}
```

Comment:
See #125298 for the reasonning behind using `int` for `self.usercodes` type hints.